deploy/vagrant_xenial_gui/Vagrantfile

```diff
@@ -5,13 +5,17 @@ Vagrant.configure("2") do |config|
 
   config.vm.box = "NathanielPaulus/xenial64-xforge"
 
+  config.vm.hostname = "vag-u1604-xf"
+
   config.vm.provider "virtualbox" do |vb|
+    vb.name="vag-u1604-xf"
     vb.gui = true
-    vb.memory = "2048"
+    vb.cpus = 4
+    vb.memory = "4096"
```

Comment:
First I had to reduce the CPUs and RAM to get it to work at all on my laptop (it only has 4GB RAM and 4 CPUs). I'm not sure how wise it is to specify this so high when you don't know what machine it will be run on. A user could set those appropriately after the box is created. We would have to add that to the docs.

Reply:
We know that the e2e tests won't pass without at least 3 gigabytes of memory. Maybe your laptop doesn't meet our minimum development specs. :-)